lighthouse-core/report/html/templates.html

```diff
@@ -406,14 +406,14 @@
       </button>
       <div id="lh-tools-dropdown" role="menu" class="lh-tools__dropdown" aria-labelledby="lh-tools-button">
          <!-- TODO(i18n): localize tools dropdown -->
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--print" id="lh-dropdown__print-summary" data-action="print-summary">Print Summary</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--print" id="lh-dropdown__print-expanded" data-action="print-expanded">Print Expanded</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--copy" id="lh-dropdown__json" data-action="copy">Copy JSON</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--download" id="lh-dropdown__save-html" data-action="save-html">Save as HTML</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--download" id="lh-dropdown__save-json" data-action="save-json">Save as JSON</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--open lh-tools--viewer" id="lh-dropdown__open-viewer" data-action="open-viewer">Open in Viewer</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--open lh-tools--gist" id="lh-dropdown__save-gist" data-action="save-gist">Save as Gist</a>
-        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--dark" id="lh-dropdown__dark-theme" data-action="toggle-dark">Toggle Dark Theme</a>
+        <a role="menuitem" tabindex="-1" href="#" class="report-icon report-icon--print" data-i18n="dropdownPrintSummary" data-action="print-summary">Print Summary</a>
```

Comment:
we could probably have a test that checks all the `[data-i18n]` are good nodes using jsdom. `report-renderer-test` is probably a good place for that.

Reply:
What do you mean by "good nodes"?

Reply:
that the value of these attributes are really i18n keys.